### PR TITLE
minor changes on the ECR code to view the report more precisely

### PR DESCRIPTION
**Pull request by sainathdev**
## Proposed changes

The Earlier code would show the report with and without vulnerabilities tags on the cloudreport. 
The code is now updated with two features
1. Get the Images and the tag of those only with the vulnerabilities and does not show the TAGs/Images without issues/vulnerabilities .
2. The code also shows the tags that with same SHA's as the comma separated tags. This mean when the same SHA of the image is used on the different environments, the image tag is now counted as a new vulnerabiliy.
-->

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (changes to code, which do not change application behavior)

## Checklist

- [ ] I have filled out this PR template
- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (`README.md`, `CHANGELOG.md`, etc. - if appropriate)

### CODE REVIEW

**Comment by sainathdev** [comment]
[reportscan.log](https://github.com/Optum/cloudig/files/7455050/reportscan.log)

**Comment by sainathdev** [comment]
@bjfish25 Updated the notes and added the log report as an example. Please review. Thank you.

**Review by shubydo — APPROVED** [review]

**Comment by sainathdev** [comment]
> @sainathdev Can you update the doc page [here](https://github.com/Optum/cloudig/blob/master/doc/ecrscan.md) to include an example of a finding with an image that has more than one tag?

@shubydo  I have updated the md file. please review.

**Review by bjfish25 — CHANGES_REQUESTED** [review]
@sainathdev Can you fix your second commit, besides hat this looks good to merge

File: doc/ecrscan.md [hunk 1]
```diff
@@ -48,7 +47,7 @@ Sample output with mdtable format, with specific tag "latest"
 
 | ACCOUNT ID   | REGION    | REPOSITORY NAME                     | TAG    | VULNERABILITIES(COUNT) | COMMENTS    |
 | ------------ | --------- | ----------------------------------- | ------ | ---------------------- | ----------- |
-| 111111111111 | us-east-1 | temporary-custom-terraform-provider | latest | LOW: 6                 | NEW_FINDING |
-|              |           |                                     |        | HIGH: 18               |             |
+| 111111111111 | us-east-1 | prod-2021-09-06.01,                 | latest | LOW: 6                 | NEW_FINDING |
+|              |           | nonprod-2021-09-03.1, release-P8-1  |        | HIGH: 18               |             |
```
Comment (shubydo) [inline doc/ecrscan.md:51]: The output doesn't match here from what I get when running this locally. Each repository should have its own section, correct?

Ex:
```markdown
| ACCOUNT ID   | REGION    | REPOSITORY NAME             | TAG    | VULNERABILITIES(COUNT) | COMMENTS    |
|--------------|-----------|-----------------------------|--------|------------------------|-------------|
| 111111111111 | us-east-1 | dig/simple-server           | latest | MEDIUM:         8      | NEW_FINDING |
|              |           |                             |        | LOW:            1      |             |
|              |           |                             |        | HIGH:           2      |             |
|              |           | dig/sample-app              | latest | CRITICAL:       37     | NEW_FINDING |
|              |           |                             |        | HIGH:           67     |             |
|              |           |                             |        | MEDIUM:         164    |             |
|              |           |                             |        | LOW:            35     |             |
```

**Review by shubydo — CHANGES_REQUESTED** [review]

File: doc/ecrscan.md (hunk 1 above)
Comment (shubydo) [reply doc/ecrscan.md:51]: Also, this would be good to add as a test case [here](https://github.com/Optum/cloudig/blob/master/pkg/cloudig/output_test.go).

**Review by shubydo — APPROVED** [review]

**Review by shubydo — APPROVED** [review]

**Review by bjfish25 — APPROVED** [review]

**Pull request merged**